src/services/pool/mod.rs

```diff
@@ -578,7 +578,8 @@ impl PoolService {
         };
 
         if path.as_path().exists() {
-            return Err(PoolError::NotCreated("Already created".to_string()));
+            return Err(PoolError::AlreadyExists(
```

Comment:
> It would help to have the path that the rust library is seeing in the error message

It is internal information about libindy persistent storage. All you need to know as the developer is the pool config name, but you just passed this name to this call. 

Actually, it is a good idea to add pool config name to this error message.